noble/config.go

```diff
@@ -21,6 +21,8 @@ type ChainConfig struct {
 
 	BlockQueueChannelSize uint64 `yaml:"block-queue-channel-size"`
 
+	MinAmount uint64 `yaml:"min-amount"`
```

Comment:
maybe `min-mint-amount` for clarity that this only applies when the domain is the destination?